<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeWindow</class>
 <widget class="QMainWindow" name="ModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mode</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="QuestionLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Do you want to continue as an admin or as a user?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="UserButton">
            <property name="text">
             <string>User</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="AdminButton">
            <property name="text">
             <string>Admin</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ViewButton">
        <property name="text">
         <string>Neither, I just want to see the lists of movies.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
